Adi.Core.Api.Tests.Unit/DeleteMe.cs

```diff
@@ -0,0 +1,11 @@
+namespace Adi.Core.Api.Tests.Unit
```

Comment:
Add copyright Header
// ---------------------------------------------------------------------------------- 
// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
// ----------------------------------------------------------------------------------

This should be at the head of every file where possible, followed by the usings.